### PR TITLE
Provision of edits for Enums DM

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is my attempt to provide edits to the existing enums ref for DM.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [ / ] All writings are my own.
- [ / ] My entry follows the Codecademy Docs style guide.
- [ / ] My changes generate no new warnings.
- [ / ] I have performed a self-review of my own writing and code.
- [ / ] I have checked my entry and corrected any misspellings.
- [ / ] I have made corresponding changes to the documentation if needed.
- [ / ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ / ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ / ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by arisdelacruz** [comment]
@yangc95 

Just trying to introduce enhancements

Thanks

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz Looks good to me.

**Comment by SSwiniarski** [comment]
@arisdelacruz note I resolved the conflicts in this file.

**Comment by arisdelacruz** [comment]
@SSwiniarski 

Thanks much

File: content/discrete-math/concepts/enumeration/enumeration.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ CatalogContent:
...
+
+Some notes on `"AND"`:
+
+- corresponds to the Cartesian product of sets
+- If you choose one element of the set A AND you choose one element of the set B, then this is equivalent to choosing one element of the `Cartesian product of A and B`
+
+Some notes on `"OR"`:
+
+- corresponds to the union of sets
+- If you choose one element of the set A OR you choose one element of the set B, then this is equivalent to choosing one element of the `union of A and B`
```
Comment (yangc95) [inline content/discrete-math/concepts/enumeration/enumeration.md:33]:
```suggestion
- Corresponds to the Cartesian product of sets.
- If one element of the set A `AND` one element of the set B is chosen, then this is equivalent to choosing one element of the `Cartesian product of A and B`.

Some notes on `OR`:

- Corresponds to the union of sets.
- If an element of the set A `OR` one element of the set B is chosen, then this is equivalent to choosing one element of the union of A and B.
```

**Review by yangc95 — APPROVED** [review]
LGTM

**Pull request merged**